src/tests/functional/plugin/shared/src/behavior/compiled_model/properties.cpp

```diff
@@ -0,0 +1,344 @@
+// NOT USED
+TEST_P(OVClassCompiledModelSetCorrectConfigTest, canSetConfig) {
+    ov::Core ie = createCoreWithTemplate();
+    ov::Any param;
+
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device);
+    OV_ASSERT_NO_THROW(compiled_model.set_property({{configKey, configValue}}));
+    OV_ASSERT_NO_THROW(param = compiled_model.get_property(configKey));
+    EXPECT_FALSE(param.empty());
+    ASSERT_EQ(param, configValue);
+}
+
+TEST_P(OVClassCompiledModelSetIncorrectConfigTest, canNotSetConfigToCompiledModelWithIncorrectConfig) {
+    ov::Core ie = createCoreWithTemplate();
+
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device);
+    std::map<std::string, std::string> incorrectConfig = {{"abc", "def"}};
+    std::map<std::string, ov::Any> config;
+    for (const auto& confItem : incorrectConfig) {
+        config.emplace(confItem.first, confItem.second);
+    }
+    EXPECT_ANY_THROW(compiled_model.set_property(config));
+}
+
+
+// writeble
+
+TEST_P(OVClassCompiledModelGetPropertyTest_MODEL_PRIORITY, GetMetricNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device, configuration);
+
+    ov::hint::Priority value;
+    OV_ASSERT_NO_THROW(value = compiled_model.get_property(ov::hint::model_priority));
+    ASSERT_EQ(value, configuration[ov::hint::model_priority.name()].as<ov::hint::Priority>());
+}
+
+TEST_P(OVClassCompiledModelGetPropertyTest_DEVICE_PRIORITY, GetMetricNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device, configuration);
+
+    std::string value;
+    OV_ASSERT_NO_THROW(value = compiled_model.get_property(ov::device::priorities));
+    ASSERT_EQ(value, configuration[ov::device::priorities.name()].as<std::string>());
+}
+
+TEST_P(OVClassCompiledModelGetPropertyTest_EXEC_DEVICES, CanGetExecutionDeviceInfo) {
+    ov::Core ie = createCoreWithTemplate();
+    std::vector<std::string> expectedTargets = {expectedDeviceName};
+    auto compiled_model = ie.compile_model(model, target_device, compileModelProperties);
+
+    std::vector<std::string> exeTargets;
+    OV_ASSERT_NO_THROW(exeTargets = compiled_model.get_property(ov::execution_devices));
+
+    ASSERT_EQ(expectedTargets, exeTargets);
+}
+
+TEST_P(OVCompileModelGetExecutionDeviceTests, CanGetExecutionDeviceInfo) {
+    ov::CompiledModel exeNetWork;
+    auto deviceList = core->get_available_devices();
+    std::vector<std::string> expected_devices = util::split(expectedDeviceName, ',');
+    std::vector<std::string> updatedExpectDevices;
+    updatedExpectDevices.assign(expected_devices.begin(), expected_devices.end());
+    for (auto &iter : compileModelProperties) {
+        if ((iter.first == ov::hint::performance_mode && iter.second.as<ov::hint::PerformanceMode>() == ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT) ||
+            target_device.find("MULTI") != std::string::npos) {
```

Comment:
why only for multy? What if devices for multi is set by config instead of `Multi:cpu`?

Reply:
I can't say for sure, this test is used by several plugins and we do not use it for apiConfoamnce. For Multi:cpu it should work correct, but is it better to control this condition with skip config ?